src/technologies/h.json

```
@@ -1788,11 +1788,38 @@
     "scriptSrc": "/v1/lst/universal-script\\?ph\\=",
     "website": "https://hyros.com"
   },
+  "Hyva Checkout": {
+    "cats": [
+      41
```

Would category 6 ("Ecommerce platforms") be a better category than 41 ("Payment processors")? From a quick look at the website the payment is handled by other providers (Stripe, PayPal...etc).

```suggestion
      6
```

---

The one and only Barry Pollard does the review! And once again, I admire how fast PRs are picked up by the team.

On topic: I have a hard time finding the complete list of categories anywhere, at least in the repo. I see cat 6 is "Ecommerce" which seems fitting, so that'd be good.
I'm thinking it'd probably be good to add that to "Hyvä Themes" as well, so be included in the general ecommerce section aside from "frontend".

Will update, thank you

---

Updated with commit [Update categories on Hyvä Checkout and Theme h.json](https://github.com/HTTPArchive/wappalyzer/pull/96/commits/3ac3755ecb3e79759720471e6936f1acf2d9c56a)

---

Categories are here FYI:
https://github.com/HTTPArchive/wappalyzer/blob/main/src/categories.json